src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs

```diff
@@ -217,7 +217,7 @@ public async Task<IEnumerable<OrganisationStandardSummary>> GetOrganisationStand
                     await connection.OpenAsync();
 
                 var sqlForStandardByOrganisationId =
-                    "SELECT distinct EndPointAssessorOrganisationId as organisationId, StandardCode, EffectiveFrom, EffectiveTo, DateStandardApprovedOnRegister, ContactId "+
+                     "SELECT distinct EndPointAssessorOrganisationId as organisationId, StandardCode, EffectiveFrom, EffectiveTo, DateStandardApprovedOnRegister, ContactId, OrganisationData " +
                      "FROM [OrganisationStandard] WHERE EndPointAssessorOrganisationId = @organisationId";
                 return await connection.QueryAsync<OrganisationStandardSummary>(sqlForStandardByOrganisationId, new {organisationId});
```

Comment:
This object 'OrganisationStandardSummary' doesn't have a field for OrganisationData.  Either add it to that class, or remove it from this sql.  I favour the first unless you have a very good reason to add it.

Reply:
Agreed, it was superfluous so I removed it